java/Ice/greeter/client/src/main/java/com/zeroc/greeter/Client.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) ZeroC, Inc.
+
+package com.zeroc.greeter;
+
+import com.zeroc.Ice.Communicator;
+import com.zeroc.Ice.Util;
+import com.zeroc.greeter.VisitorCenter.GreeterPrx;
+import java.util.concurrent.CompletableFuture;
+
+class Client {
+    public static void main(String[] args)
+    {
+        // Create an Ice communicator to initialize the Ice runtime. The communicator is disposed before the program exits.
+        try(com.zeroc.Ice.Communicator communicator = com.zeroc.Ice.Util.initialize(args))
```

Comment:
Is this formatting correct? I prefer `try (` and `catch (` to distinguish with method calls.

Reply:
Maybe I should configure spotless plug-in for the demos too?